internal/driver/glfw/driver_test.go

```diff
@@ -64,47 +64,57 @@ func Test_gLDriver_AbsolutePositionForObject(t *testing.T) {
 	m2 := mbarCont.Objects[1]
 
 	tests := map[string]struct {
-		object fyne.CanvasObject
-		want   fyne.Position
+		object       fyne.CanvasObject
+		wantX, wantY int
```

Comment:
any particular reason about this change?

Reply:
Yes, the positions are no longer round whole numbers, so comparing the int form is easier

Reply:
I see, got it :)